ui/page/claim/internal/claimPageComponent/internal/channelPage/tabs/contentTab/view.jsx

```diff
@@ -104,12 +106,17 @@ function ContentTab(props: Props) {
 
   function handleInputChange(e) {
     const { value } = e.target;
+
     setSearchQuery(value);
   }
 
+  React.useEffect(() => {
+    setOrderBy(urlParams);
+  }, [search, urlParams]);
```

Comment:
Hm ... I still think there is no need for `order` to be a state, unless I missed something, like needing it to cross `undefined --to--> defined` in two cycles.  

`search` is already somewhat a state from `useHistory`, so it will cause a render when changed, which we can immediately calculate urlParams and orderBy without listening to it through an effect.

```
const urlParams = new URLSearchParams(search).get('order');
const orderBy = urlParams;
```